src/display/lv_display.c

```diff
@@ -492,8 +492,9 @@ lv_obj_t * lv_display_get_screen_prev(lv_display_t * disp)
     return disp->prev_scr;
 }
 
-void lv_display_load_scr(lv_obj_t * scr)
+void lv_display_load_screen(lv_display_t * disp, lv_obj_t * scr)
```

Comment:
Need to add to `lv_api_map.h`

Reply:
Finally I removed this function as loading a screen is not related to the display. `lv_screen_load()` can be used directly.